content/kubeone/main/guides/cis-benchmarking/cis1.8-kubeone1.7-k8s1.27/_index.en.md

```diff
@@ -0,0 +1,574 @@
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.11: Ensure that the etcd data directory permissions are set to 700 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.12: Ensure that the etcd data directory ownership is set to etcd:etcd (Automated)
+
+**Result:** 🟢 Pass
+
+**Details:** KubeOne runs etcd cluster as containers and there is not a `etcd` user and group on the system
+
+---
+#### 1.1.13: Ensure that the admin.conf file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.14: Ensure that the admin.conf file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.15: Ensure that the scheduler.conf file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.16: Ensure that the scheduler.conf file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.17: Ensure that the controller-manager.conf file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.18: Ensure that the controller-manager.conf file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.19: Ensure that the Kubernetes PKI directory and file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.20: Ensure that the Kubernetes PKI certificate file permissions are set to 600 or more restrictive (Manual)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.21: Ensure that the Kubernetes PKI key file permissions are set to 600 (Manual)
+
+**Result:** 🟢 Pass
+
+---
+### 1.2. API Server
+#### 1.2.1: Ensure that the --anonymous-auth argument is set to false (Manual)
+
+**Result:** 🔴 Fail
+
+_The issue is under investigation to provide a fix in a future KubeOne release_
+
+---
+#### 1.2.2: Ensure that the --token-auth-file parameter is not set (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.3: Ensure that the --DenyServiceExternalIPs is set (Manual)
+
+**Result:** 🔴 Fail
+
+_The issue is under investigation to provide a fix in a future KubeOne release_
+
+---
+#### 1.2.4: Ensure that the --kubelet-client-certificate and --kubelet-client-key arguments are set as appropriate (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.5: Ensure that the --kubelet-certificate-authority argument is set as appropriate (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.6: Ensure that the --authorization-mode argument is not set to AlwaysAllow (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.7: Ensure that the --authorization-mode argument includes Node (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.8: Ensure that the --authorization-mode argument includes RBAC (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.2.9: Ensure that the admission control plugin EventRateLimit is set (Manual)
+
+**Result:** 🟢 Pass
+
+**Details:** EventRateLimit admission control plugin in in Alpha state, please see [here][eventratelimit]
```

Comment:
I was thinking more about this audit and I think it's actually a fail. We don't enable this admission control plugin, CIS requires it and doesn't go into details if it's alpha or not. It's up to us to decide if we want to support an alpha feature or not (we have always been towards not supporting alpha features and we can check with @csengerszabo if it's safe to state so in this document)